src/Microsoft.AspNet.Authentication/AuthenticationOptions.cs

```diff
@@ -27,6 +27,13 @@ public string AuthenticationScheme
         }
 
         /// <summary>
+        /// If true the authentication middleware alter the request user coming in and
+        /// alter 401 Unauthorized responses going out. If false the authentication middleware will only provide
+        /// identity and alter responses when explicitly indicated by the AuthenticationScheme.
+        /// </summary>
+        public bool AutomaticAuthentication { get; set; }
```

Comment:
Is it worth resurrecting `AuthenticationMode`? (with `Active` -> `Automatic` and `Passive` -> `Manual`)?